packages/backend/src/lib/git-instance.ts

```diff
@@ -42,6 +42,12 @@ export type ApplyGitChangesArgs = {
   commitMessage: string;
 };
 
+export type RollBackCommitArgs = {
```

Comment:
I don't typically use this args pattern in the backend, just in the React code where it works well with the JSX notation.  

Typically this pattern is also useful if the function took **a lot** of arguments, including optional arguments, as it would make it more readable than a long line of comments.  But in this case, there's no advantage over just adding 3 arguments to the function directly.